graphein/ml/datasets/foldcomp_dataset.py

```diff
@@ -411,9 +430,12 @@ def _compose_transforms(self, transforms: Iterable[Callable]) -> T.Compose:
             return T.Compose(transforms)
 
     def setup(self, stage: Optional[str] = None):
-        self.train_dataset()
-        self.val_dataset()
-        self.test_dataset()
+        if stage == "fit":
```

Comment:
Can we add a default case when `stage=None`?

Reply:
Added that case now where we do all three processing stages.